utility/README.md

```diff
@@ -1,679 +1,1344 @@
+A `RelayChain` is an identifier of the specified Web3 data source (i.e. a blockchain) being interacted with for that session.
 
+For example, `0021` represents `Ethereum Mainnet` in Pocket Network V0.
 
-```
-func IsSalaryEligible(ReportCard, RCThreshold, TSThreshold) Bool {
-  If ReportCard.Score < RCThreshold { Return False } 
-  If ReportCard.Samples < TSThreshold { Return False }
-  Return True
-} 
+#### 3.1.3 GeoZone
 
-func GetSalary(ReportCard, StakeAmountScore, TotalAvailReward, EligibleNodesCount) Salary {
-  # calculate max reward per ServiceNode
-  maxServiceNodeReward = perfectDivide(TotalAvailReward, EligibleNodesCount)
-  # calculate decrease (NOTE: likely will weight each score with a coefficient param)
-  burnPercent = (100-ReportCard.Score + 100-StakeAmountScore)/200
-  burnAmount = burnPercent * maxServiceNodeReward
-  netServiceNodeReward = maxServiceNodeReward - burnAmount
-  burnTokensFromTotalAvailableRewardPool(burnAmount)
-  Return netServiceNodeReward
-} 
-```
+A `GeoZone` is a representation of a physical geo-location the actors advertise they are in.
 
+For example, `GeoZone 0001` could represent `US East`, but alternative coordinate systems such as [Uber's H3](https://h3geo.org) or [PostGIS](https://postgis.net/) could be used as well.
 
-In Pocket 1.0 ServiceNodes are able to use a PauseMsg to gracefully remove them from service. This feature allows for greater UX for Node Runners, enabling scheduled periods of maintenance or damage control in faulty situations. In addition to an Operator initiated PauseMsg, Fishermen are able to pause a ServiceNode if a faulty or malicious process is detected during sampling, the details, limitations, and incentives of which are described in the Fisherman Protocol section. If a ServiceNode is ‘paused’, they are able to reverse the paused state by submitting an UnpauseMsg after the MinSNPauseTime has elapsed. After a successful UnpauseMsg, the ServiceNode is once again eligible to receive Web3 traffic from Applications and Fisherman. 
+There is no formal requirement or validation (e.g. IP verification) for an actor to be physically located in the GeoZone it registers in. However, crypto-economic incentives drive actors to be registered close to where they are physically located to receive and provide the best service possible.
 
+The number of GeoZones andy actor can stake for is limited to one to incentivize real geographic distribution. To quote @deblasis: "If an actor is everywhere, they're nowhere."
```

Comment:
"andy"? -> "any"?

Reply:
What is _real_ geographic distribution? Given that there exists a `ServiceUrl` that eventually resolves into an IP address, this goes against [AnyCast](https://en.wikipedia.org/wiki/Anycast). Why do we seek to limit it?

Interesting change for when regenesis happens from V0 to V1. Limiting geographic zones definitely is more simplistic, however in practicality, probably not. Node runners will converge where the economic incentives are. Pocket V1 allows for lightweight servicers, so what you end up creating is an incentive to hop to the highest traffic region by transferring keys from one server to another, effectively creating a regional hotspot now and then.

I'll need to think about further implications, but definitely, one of those tops of mind is arbitrarily creating regional hotspots and making it hard to bootstrap more PoPs like a "central" / new geo zone without having more Pocket at stake.

Reply:
> What is real geographic distribution?

You are correct that there is no way to enforce a real physical location (tor, VPN, etc...), but the best we can do (and what crypto is about) is incentivize it.

> Why do we seek to limit it?

I see it as incentivizing literal physical geo-distribution of servers to have precedence of being matched with applications in some region. If a certain country starts banning web3 access, the physical location of a server could play an important role in decentralization.

> I'll need to think about further implications, but definitely, one of those tops of mind is arbitrarily creating regional hotspots and making it hard to bootstrap more PoPs like a "central" / new geo zone without having more Pocket at stake.

Agreed that there's a lot to think through an unpack here, and free-market economics could lead to interesting solutions. Happy to review a change to the spec in a future iteration when you have an interesting proposal.